README.md

```diff
@@ -1,63 +1,128 @@
+<p align="center">
+    <a href="http://vapor.codes">
+        <img src="https://img.shields.io/badge/Vapor-2.x-blue.svg" alt="Vapor">
+    </a>
+    <a href="https://github.com/SwiftyBeaver/SwiftyBeaver">
+        <img src="https://img.shields.io/badge/SwiftyBeaver-1.x-blue.svg" alt="SwiftyBeaver">
+    </a>
+    <!--a href="https://slack.swiftybeaver.com">
+        <img src="https://slack.swiftybeaver.com/badge.svg" alt="Slack Team">
+    </a-->
+    <a href="LICENSE">
+        <img src="http://img.shields.io/badge/license-MIT-brightgreen.svg" alt="MIT License">
+    </a>
+    <!--a href="https://circleci.com/gh/SwiftyBeaver/SwiftyBeaver-Vapor">
+        <img src="https://circleci.com/gh/SwiftyBeaver/SwiftyBeaver-Vapor.svg?style=shield" alt="Continuous Integration">
+    </a>
+    <a href="https://travis-ci.org/SwiftyBeaver/SwiftyBeaver-Vapor">
+    	<img src="https://travis-ci.org/SwiftyBeaver/SwiftyBeaver-Vapor.svg?branch=master" alt="Build Status">
+    </a-->
+    <a href="https://swift.org">
+        <img src="http://img.shields.io/badge/swift-3.1-brightgreen.svg" alt="Swift 3.1">
+    </a>
+</center>
+
 # SwiftyBeaver Logging Provider for Vapor
-[![Language Swift 3](https://img.shields.io/badge/Language-Swift%203-orange.svg)](https://swift.org) [![Vapor 1.x](https://img.shields.io/badge/Vapor-1.x-blue.svg)](http://vapor.codes/) [![SwiftyBeaver 1.x](https://img.shields.io/badge/SwiftyBeaver-1.x-blue.svg)] (https://github.com/SwiftyBeaver/SwiftyBeaver) [![Slack Status](https://slack.swiftybeaver.com/badge.svg)](https://slack.swiftybeaver.com) 
 
 Adds the powerful logging of [SwiftyBeaver](https://github.com/SwiftyBeaver/SwiftyBeaver) to [Vapor](https://github.com/vapor/vapor) for server-side Swift 3 on Linux and Mac.
 
 ## Installation
 
-Add this to the `Package.swift` of your Vapor project:
+Add this project to the `Package.swift` of your Vapor project:
 
 ```swift
-dependencies: [
-	.Package(url: "https://github.com/SwiftyBeaver/SwiftyBeaver-Vapor.git", majorVersion: 1),
-	//...other packages here
-],
+import PackageDescription
+
+let package = Package(
+    name: "Project",
+    dependencies: [
+        .Package(url: "https://github.com/vapor/vapor.git", majorVersion: 2),
+        .Package(url: "https://github.com/SwiftyBeaver/SwiftyBeaver-Vapor.git", majorVersion: 2),
```

Comment:
While I see you want to provide context for where this line should go I think you really only need what was originally there changed to version 2 of SwiftyBeaver-Vapor.

```
dependencies: [
	.Package(url: "https://github.com/SwiftyBeaver/SwiftyBeaver-Vapor.git", majorVersion: 1),		 
 	//...other packages here
 ],